.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] The `.gitignore` file should also contain a `test.md` file that has to be created in your local environment but ignored by this file.